src/daemon/ca/rc.rs

```diff
@@ -301,40 +301,20 @@ impl ResourceClass {
                 rcvd_cert.validity().not_after().to_rfc3339()
             );
 
+            // Prep certified key for updated received certificate
+            let certified_key = CertifiedKey::create(rcvd_cert);
+
             // Check whether child certificates should be shrunk
```

Comment:
Based on the name of the invoked function and the modification made to the descriptive comment on the function, I assume this comment here is incorrect: this doesn't just check if shrinking is needed, it will also actually do the shrinking, right?

Reply:
the logic was correct, renamed things and comments to make it more clear.